File: selenium/java/example-junit/src/test/java/tests/TestAuthorization2.java
package tests;

import org.junit.jupiter.api.AfterAll;
import org.junit.jupiter.api.AfterEach;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.BeforeAll;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Disabled;
import org.junit.jupiter.api.Tag;
import org.junit.jupiter.api.Test;

import org.openqa.selenium.chrome.ChromeDriver;

import support.PageObjects.CommonPage;
import support.StepObjects.CommonSteps;
import support.Helper;

public class TestAuthorization2 {

    public static CommonSteps tester;

    @BeforeAll
    public static void before(){
        tester = new CommonSteps(new ChromeDriver());
        tester.driver.manage().window().maximize();
    }

    @Tag("PROD")
    @Test
    public void testAuthorization(){
        tester.driver.get(Helper.URL);
        tester.sendForm(Helper.LOGIN, Helper.PASSWORD);
        String text = CommonPage.getResultText(tester.driver);
        Assertions.assertEquals(text, "Authorization was successful");
    }

    @AfterAll
    public static void after(){
        tester.driver.close();
        tester.driver.quit();
    }
}

File: selenium/python/example-unittest/tests/TestAuthorization2.py
import unittest

from selenium import webdriver
from support.PageObjects.CommonPage import CommonPage
from support.StepObjects.CommonSteps import CommonSteps

class TestAuthorization2(unittest.TestCase):
    def test(self):
        self.driver = webdriver.Chrome()
        self.driver.maximize_window()
        tester = CommonSteps(self.driver)
        tester.driver.get('https://somovstudio.github.io/test_eng.html')
        tester.sendForm('admin', '0000')
        text = CommonPage.getResultText(tester.driver)
        self.assertEqual(text, 'Authorization was successful')
        self.driver.close()
        self.driver.quit()

if __name__ == '__main__':
    unittest.main()
